salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc

```diff
@@ -42,9 +42,18 @@ You must know the scope of the rate limiting policy that applies to your account
 
 When extracting data from a Salesforce Opportunity object, be aware that a "quarter" in this context is relative to the financial year of the organization and not necessarily to the calendar year.
 
+== Japanese Support
+
+Salesforce Connector supports Japanese characters for most operations, with these exceptions:
```

Comment:
Suggestion: 
"Salesforce Connector supports Japanese characters for all operations, with these exceptions:"
or
"Salesforce Connector supports Japanese characters for all operations, except for the following:"

Reply:
I don't think I should change it to all. They may not have tested all operations to make sure that's the case. Alex stated most in his original wording so I think it's safer to keep that.